From f35de48400350d5194ab828cc0cab10b33207a57 Mon Sep 17 00:00:00 2001
From: Liu Ying <victor.liu@nxp.com>
Date: Thu, 21 Mar 2019 13:43:15 +0800
Subject: [PATCH 1/5] MLK-21123-1 arm64: fsl-imx8qm-device.dtsi: Correct DPU
 endpoint names

commit 742d572d8f3b9be063adb1d29fbf16907af5fbfe from
https://source.codeaurora.org/external/imx/linux-imx.git

This patch corrects DPU endpoint names to suppress DTC warnings like below:
Warning (graph_endpoint): /dpu@56180000/port@0/hdmi-endpoint: graph endpont node name should be 'endpoint'
Warning (graph_endpoint): /dpu@56180000/port@0/mipi-dsi-endpoint: graph endpont node name should be 'endpoint'
Warning (graph_endpoint): /dpu@56180000/port@1/lvds0-endpoint: graph endpont node name should be 'endpoint'
Warning (graph_endpoint): /dpu@56180000/port@1/lvds1-endpoint: graph endpont node name should be 'endpoint'
Warning (graph_endpoint): /dpu@57180000/port@0/mipi-dsi-endpoint: graph endpont node name should be 'endpoint'
Warning (graph_endpoint): /dpu@57180000/port@1/lvds0-endpoint: graph endpont node name should be 'endpoint'
Warning (graph_endpoint): /dpu@57180000/port@1/lvds1-endpoint: graph endpont node name should be 'endpoint'

Signed-off-by: Liu Ying <victor.liu@nxp.com>
Signed-off-by: Zhixiong Chi <zhixiong.chi@windriver.com>
---
 .../boot/dts/freescale/fsl-imx8qm-device.dtsi      | 14 +++++++-------
 .../dts/freescale/fsl-imx8qm-lpddr4-arm2-domu.dts  |  6 +++---
 .../boot/dts/freescale/fsl-imx8qm-mek-domu.dts     |  6 +++---
 3 files changed, 13 insertions(+), 13 deletions(-)

diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8qm-device.dtsi b/arch/arm64/boot/dts/freescale/fsl-imx8qm-device.dtsi
index d7f9eb74971c..2c72ffd76f00 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8qm-device.dtsi
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8qm-device.dtsi
@@ -1774,11 +1774,11 @@
 		dpu1_disp0: port@0 {
 			reg = <0>;
 
-			dpu1_disp0_hdmi: hdmi-endpoint {
+			dpu1_disp0_hdmi: endpoint@0 {
 				remote-endpoint = <&hdmi_disp>;
 			};
 
-			dpu1_disp0_mipi_dsi: mipi-dsi-endpoint {
+			dpu1_disp0_mipi_dsi: endpoint@1 {
 				remote-endpoint = <&mipi_dsi1_in>;
 			};
 		};
@@ -1786,11 +1786,11 @@
 		dpu1_disp1: port@1 {
 			reg = <1>;
 
-			dpu1_disp1_lvds0: lvds0-endpoint {
+			dpu1_disp1_lvds0: endpoint@0 {
 				remote-endpoint = <&ldb1_lvds0>;
 			};
 
-			dpu1_disp1_lvds1: lvds1-endpoint {
+			dpu1_disp1_lvds1: endpoint@1 {
 				remote-endpoint = <&ldb1_lvds1>;
 			};
 		};
@@ -2248,7 +2248,7 @@
 		dpu2_disp0: port@0 {
 			reg = <0>;
 
-			dpu2_disp0_mipi_dsi: mipi-dsi-endpoint {
+			dpu2_disp0_mipi_dsi: endpoint {
 				remote-endpoint = <&mipi_dsi2_in>;
 			};
 		};
@@ -2256,11 +2256,11 @@
 		dpu2_disp1: port@1 {
 			reg = <1>;
 
-			dpu2_disp1_lvds0: lvds0-endpoint {
+			dpu2_disp1_lvds0: endpoint@0 {
 				remote-endpoint = <&ldb2_lvds0>;
 			};
 
-			dpu2_disp1_lvds1: lvds1-endpoint {
+			dpu2_disp1_lvds1: endpoint@1 {
 				remote-endpoint = <&ldb2_lvds1>;
 			};
 		};
diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8qm-lpddr4-arm2-domu.dts b/arch/arm64/boot/dts/freescale/fsl-imx8qm-lpddr4-arm2-domu.dts
index 0c6e5156bd38..82f4cfaa0497 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8qm-lpddr4-arm2-domu.dts
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8qm-lpddr4-arm2-domu.dts
@@ -260,18 +260,18 @@
 	status = "okay";
 
 	dpu2_disp0: port@0 {
-		dpu2_disp0_mipi_dsi: mipi-dsi-endpoint {
+		dpu2_disp0_mipi_dsi: endpoint {
 			/delete-property/ remote-endpoint;
 		};
 	};
 	dpu2_disp1: port@1 {
 		reg = <1>;
 
-		dpu2_disp1_lvds0: lvds0-endpoint {
+		dpu2_disp1_lvds0: endpoint@0 {
 			remote-endpoint = <&ldb2_lvds0>;
 		};
 
-		dpu2_disp1_lvds1: lvds1-endpoint {
+		dpu2_disp1_lvds1: endpoint@1 {
 			remote-endpoint = <&ldb2_lvds1>;
 		};
 	};
diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8qm-mek-domu.dts b/arch/arm64/boot/dts/freescale/fsl-imx8qm-mek-domu.dts
index 5b10d03140b6..2a31da522f2e 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8qm-mek-domu.dts
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8qm-mek-domu.dts
@@ -511,18 +511,18 @@
 	status = "okay";
 
 	dpu2_disp0: port@0 {
-		dpu2_disp0_mipi_dsi: mipi-dsi-endpoint {
+		dpu2_disp0_mipi_dsi: endpoint {
 			/delete-property/ remote-endpoint;
 		};
 	};
 	dpu2_disp1: port@1 {
 		reg = <1>;
 
-		dpu2_disp1_lvds0: lvds0-endpoint {
+		dpu2_disp1_lvds0: endpoint@0 {
 			remote-endpoint = <&ldb2_lvds0>;
 		};
 
-		dpu2_disp1_lvds1: lvds1-endpoint {
+		dpu2_disp1_lvds1: endpoint@1 {
 			remote-endpoint = <&ldb2_lvds1>;
 		};
 	};
-- 
2.17.1

